src/util/isArrayLike.ts

```diff
@@ -0,0 +1 @@
+export const isArrayLike = (<T>(x: any): x is ArrayLike<T> => x && typeof x.length === 'number');
```

Comment:
if it contains multiple lines I'd definitely agree, in this case would like to follow suggestion in https://github.com/ReactiveX/RxJS/pull/1211#issuecomment-172620017 , similar case to this.